packages/react-admin-core/src/DeleteMutation.tsx

```diff
@@ -1,10 +1,5 @@
 import { useApolloClient } from "@apollo/react-hooks";
-import Button from "@material-ui/core/Button";
-import Dialog from "@material-ui/core/Dialog";
-import DialogActions from "@material-ui/core/DialogActions";
-import DialogContent from "@material-ui/core/DialogContent";
-import DialogContentText from "@material-ui/core/DialogContentText";
-import DialogTitle from "@material-ui/core/DialogTitle";
+import { Button, Dialog, DialogActions, DialogTitle, Typography } from "@material-ui/core";
```

Comment:
Ich finde path-imports, so wie sie jetzt sind, besser. Dann haben wir auch in dev-environment und ohne tree-shaking ein kleine bundle-size.